staging/src/k8s.io/client-go/tools/cache/reflector.go

```diff
@@ -185,10 +188,13 @@ func (r *Reflector) ListAndWatch(stopCh <-chan struct{}) error {
 	klog.V(3).Infof("Listing and watching %v from %s", r.expectedTypeName, r.name)
 	var resourceVersion string
 
-	// Explicitly set "0" as resource version - it's fine for the List()
-	// to be served from cache and potentially be delayed relative to
-	// etcd contents. Reflector framework will catch up via Watch() eventually.
-	options := metav1.ListOptions{ResourceVersion: "0"}
+	// Explicitly set resource version to have it list from cache for
```

Comment:
move this comment to where we set "0" in NewNamedReflector? LastSyncResourceVersion() here isn't necessarily listing from cache